sketch_oct9a.ino

```diff
@@ -120,35 +120,8 @@ void updateSDCardLEDs() {
   digitalWrite(5, good);
 }
 
-void setup() {
-  Serial1.begin(9600);
-
-#if CAPSULE == 2
-  // Pin A6: analog input from VOC sensor
-  pinMode(A6, PinMode::INPUT);
-#endif
-  // Pin D5: SD LEDs
-  pinMode(5, PinMode::OUTPUT);
-  // Pin D6: GPS LEDs
-  pinMode(6, PinMode::OUTPUT);
-  // Pin D7: sensor LEDs
-  pinMode(7, PinMode::OUTPUT);
-
-  pinPeripheral(0, PIO_SERCOM);
-  pinPeripheral(1, PIO_SERCOM);
-  pinPeripheral(8, PIO_SERCOM);
-  pinPeripheral(9, PIO_SERCOM);
-  pinPeripheral(A1, PIO_SERCOM);
-  pinPeripheral(A2, PIO_SERCOM);
-  pinPeripheral(A3, PIO_SERCOM);
-  pinPeripheral(A4, PIO_SERCOM);
-
-  // Turn on the error LEDs until initialization finishes
-  digitalWrite(5, LOW);
-  digitalWrite(6, LOW);
-  digitalWrite(7, LOW);
-
-  // Initialize sensors
+// Try to initialize all sensors. Has no effect once everything is initialized.
+void initializeAll() {
```

Comment:
I think you accidentally pulled in the changes from the other branch -- can you remove those commits?